pallets/claims/src/lib.rs

```diff
@@ -37,6 +36,7 @@ pub trait Config: frame_system::Config {
 	type Currency: Currency<Self::AccountId>;
 	type Prefix: Get<&'static [u8]>;
 	type WeightInfo: WeightInfo;
+	type IntoBalance: From<u128> + Into<<Self::Currency as Currency<<Self as frame_system::Config>::AccountId>>::Balance>;
```

Comment:
I would suggest to use like this :

type CurrencyBalance: Into<BalanceOf<Self>> + From<Balance>;

and order it so Currency and CurrencyBalance is together , and make a comment that this is the currency used in Currency and needs to support conversion to Balance (as we use it directly for the actual data )